packages/arcgis-rest-feature-service/src/helpers.ts

```diff
@@ -741,6 +742,8 @@ export interface ILayerDefinition extends IHasZM {
   supportsAttachmentsByUploadId?: boolean;
   /** Boolean value indicating whether the layer supports the Calculate REST operation when updating features. */
   supportsCalculate?: boolean;
+  /** Boolean value indicating whether the layer supports exceedsLimit in a query operation. */
+  supportsExceedsLimitStatistics: boolean;
```

Comment:
Needs to be optional since this might not exist.

```suggestion
  supportsExceedsLimitStatistics?: boolean;
```